[2021-03-15 15:36:21,641] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: PostgresOperator_dag.insert_row 2021-03-15T15:36:20.427905+00:00 [queued]>
[2021-03-15 15:36:21,660] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: PostgresOperator_dag.insert_row 2021-03-15T15:36:20.427905+00:00 [queued]>
[2021-03-15 15:36:21,660] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-15 15:36:21,661] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-15 15:36:21,662] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-15 15:36:21,669] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): insert_row> on 2021-03-15T15:36:20.427905+00:00
[2021-03-15 15:36:21,674] {standard_task_runner.py:52} INFO - Started process 53 to run task
[2021-03-15 15:36:21,678] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'PostgresOperator_dag', 'insert_row', '2021-03-15T15:36:20.427905+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmp3_jwbc6p', '--error-file', '/tmp/tmp2d2ykks2']
[2021-03-15 15:36:21,680] {standard_task_runner.py:77} INFO - Job 7: Subtask insert_row
[2021-03-15 15:36:21,724] {logging_mixin.py:104} INFO - Running <TaskInstance: PostgresOperator_dag.insert_row 2021-03-15T15:36:20.427905+00:00 [running]> on host 813d4b64061a
[2021-03-15 15:36:21,767] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=PostgresOperator_dag
AIRFLOW_CTX_TASK_ID=insert_row
AIRFLOW_CTX_EXECUTION_DATE=2021-03-15T15:36:20.427905+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-15T15:36:20.427905+00:00
[2021-03-15 15:36:21,768] {postgres.py:69} INFO - Executing: INSERT INTO user_table VALUES(%s, %s, %s)
[2021-03-15 15:36:21,778] {base.py:74} INFO - Using connection to: id: postgresql. Host: db, Port: 5432, Schema: airflow, Login: postgres, Password: XXXXXXXX, extra: None
[2021-03-15 15:36:21,782] {dbapi.py:180} INFO - Running statement: INSERT INTO user_table VALUES(%s, %s, %s), parameters: (58597273, 'felix_airflow@works.com', True)
[2021-03-15 15:36:21,783] {dbapi.py:186} INFO - Rows affected: 1
[2021-03-15 15:36:21,795] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=PostgresOperator_dag, task_id=insert_row, execution_date=20210315T153620, start_date=20210315T153621, end_date=20210315T153621
[2021-03-15 15:36:21,823] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-15 15:36:21,855] {local_task_job.py:146} INFO - Task exited with return code 0
